src/__test__/e2e/signing/determinism.test.ts

```diff
@@ -26,584 +24,585 @@ import {
   gpErrors,
   jobTypes,
 } from '../../utils/helpers';
-import { initializeSeed } from '../../utils/initializeClient';
+import { initializeClient, initializeSeed } from '../../utils/initializeClient';
 import { runTestCase } from '../../utils/runners';
 let seed: Buffer;
+let client;
```

Comment:
I don't think we need to define `client` outside of the scope of the `describe` block, do we? Other tests just define it as a `const` within the `describe` block.

Reply:
Yeah you're right. Just didn't want to lose scope. Will change